app/code/Magento/InventoryCatalog/Model/ResourceModel/GetProductTypesBySkus.php

```diff
@@ -35,23 +40,39 @@ public function __construct(
      */
     public function execute(array $skus): array
     {
+        $result = [];
+
+        foreach ($skus as $index => $sku) {
+            if (!isset($this->typeBySku[$sku])) {
+                continue;
+            }
+
+            unset($skus[$index]);
+            $result[$sku] = $this->typeBySku[$sku];
+        }
+
+        if (empty($skus)) {
+            return $result;
+        }
+
         $connection = $this->resource->getConnection();
         $productTable = $this->resource->getTableName('catalog_product_entity');
 
         $select = $connection->select()
-            ->from(
-                $productTable,
-                [ProductInterface::SKU, ProductInterface::TYPE_ID]
-            )->where(
-                ProductInterface::SKU . ' IN (?)',
+             ->from(
+                 $productTable,
+                 [ProductInterface::SKU, ProductInterface::TYPE_ID]
+             )->where(
+ProductInterface::SKU . ' IN (?)',
```

Comment:
looks like there are some issues with indentation here

Reply:
@maghamed Could you point me to the correct way to setup phpcs with Magento 2? There's documentation on MEQP, but that focuses on 3rd party extensions (right?).